qgis-app/settings_docker.py

```diff
@@ -25,11 +25,11 @@
 # Don't put anything in this directory yourself; store your static files
 # in apps' "static/" subdirectories and in STATICFILES_DIRS.
 # Example: "/var/www/example.com/static/"
-STATIC_ROOT = os.environ.get("STATIC_ROOT", "/home/web/static")
+STATIC_ROOT = os.environ.get("STATIC_ROOT", "/home/web/static/static/")
 
 # URL prefix for static files.
 # Example: "http://example.com/static/", "http://static.example.com/"
-STATIC_URL = "/static/"
+STATIC_URL = "/static/static/"
```

Comment:
perhaps also use environment variable here, and make '/static/' as default

Reply:
Thanks for the review @dimasciput , I've made the change.